app/grants/templates/grants/components/hidden_inputs.html

```diff
@@ -0,0 +1,19 @@
+<input type="hidden" id="grant_id" name="grant_id" :value="grant.id">
+<input type="hidden" id="grant_slug" name="grant_slug" :value="grant.slug">
+<input type="hidden" id="grant_url" name="grant_url" :value="grant.url">
+<input type="hidden" id="grant_title" name="grant_title" :value="grant.title">
+<input type="hidden" id="grant_contract_version" name="grant_contract_version" :value="grant.contract_version">
+<input type="hidden" id="grant_contract_address" name="grant_contract_address" :value="grant.contract_address">
+<input type="hidden" id="grant_token_symbol" name="grant_token_symbol" :value="grant.token_symbol">
+<input type="hidden" id="grant_admin_address" name="grant_admin_address" :value="grant.admin_address">
+<input type="hidden" id="grant_token_address" name="grant_token_address" :value="grant.token_address">
+<input type="hidden" id="grant_logo" name="grant_logo" :value="grant.logo_url">
+<input type="hidden" id="grant_clr_prediction_curve" name="grant_clr_prediction_curve" :value="grant.clr_prediction_curve">
+<input type="hidden" id="grant_image_css" name="grant_image_css" :value="grant.image_css">
+<input type="hidden" id="is_clr_eligible" name="is_clr_eligible" :value="grant.is_clr_eligible ? grant.is_clr_eligible : ''">
+<input type="hidden" id="grant_donation_amount" name="grant_donation_amount" :value="amount" v-if="grant.amount">
+
```

Comment:
Actually i'll remove this when i'll implement this issue. https://github.com/gitcoinco/web/issues/7164 because i need to refactor how items are added to the cart.